source/extensions/loop.c

```diff
@@ -93,77 +93,91 @@ void osl_loop_idump(FILE * file, osl_loop_p loop, int level) {
   size_t i;
 
   // Go to the right level.
-  for (j = 0; j < level; j++)
+  for (j = 0; j < level; j++) {
```

Comment:
Could you please add some motivation as to _why_ you propose these stylistic changes, and in particular why _only_ in this file. For example, here is another place that does not use braces for non-compound statements after control flow cosntructs https://github.com/periscop/openscop/blob/master/source/relation.c#L168 and there are many more.  I'm afraid we don't have a conding guidelines file, but optional braces for simple statements was one of the things I inferred from code.

Reply:
The documentation specifies the code has to be written according to the Google C++ Coding Style for what can apply to C.

It states for conditions that "In general, curly braces are not required for single-line statements, but they are allowed if you like them". So I suggest not to add braces for one liner statements.

Reply:
I prefer to always add braces but that's also personal I guess. Since it is not subject to the coding style, I will revert it.